pkg/apiserver/resthandler.go

```diff
@@ -736,6 +737,10 @@ func setListSelfLink(obj runtime.Object, req *restful.Request, namer ScopeNamer)
 }
 
 func getPatchedJS(contentType string, originalJS, patchJS []byte, obj runtime.Object) ([]byte, error) {
+	// Remove "; charset=" if included in header.
```

Comment:
I guess we can leave it for now, since it merged :)

On Thu, Oct 1, 2015 at 5:12 PM, Derek Parker <notifications@github.com>
wrote:

> In pkg/apiserver/resthandler.go
> <https://github.com/kubernetes/kubernetes/pull/14874#discussion_r40981753>
> :
>
> > @@ -736,6 +737,10 @@ func setListSelfLink(obj runtime.Object, req *restful.Request, namer ScopeNamer)
> >  }
> >
> >  func getPatchedJS(contentType string, originalJS, patchJS []byte, obj runtime.Object) ([]byte, error) {
> > +	// Remove "; charset=" if included in header.
>
> I think if anything a TODO would be fine. Do you want me to add one before
> it gets merged, or leave it for now?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/14874/files#r40981753>.
>